internal/venice-test-common/src/integrationTest/java/com/linkedin/venice/controller/TestClusterLevelConfigForActiveActiveReplication.java

```diff
@@ -87,6 +89,11 @@ public void testClusterLevelActiveActiveReplicationConfigForNewHybridStores() th
         storeNameHybrid,
         new UpdateStoreQueryParams().setHybridRewindSeconds(1000L).setHybridOffsetLagThreshold(1000L));
     assertTrue(veniceAdmin.getStore(clusterName, storeNameHybrid).isActiveActiveReplicationEnabled());
+    DataReplicationPolicy drp =
+        veniceAdmin.getStore(clusterName, storeNameHybrid).getHybridStoreConfig().getDataReplicationPolicy();
+    assertNotNull(drp);
+    System.out.println(drp);
+    assertEquals(drp, DataReplicationPolicy.NON_AGGREGATE);
```

Comment:
Why is this `NON_AGGREGATE`?

Reply:
Actually, I'm confused. Looks like it only creates a single region setup. This test doesn't make sense. What does A/A mean in single region?

Reply:
`NON_AGGREGATE`  is default policy when converting a store to hybrid mode. 
A/A DR doesn't mean anything in a single region setup IMO

Reply:
Yeah, I was actually referring to the fact that `TestClusterLevelConfigForActiveActiveReplication` has an incorrect setup. AA doesn't mean anything in single-colo setups. It's alright to not address this in your PR